packages/optimistic-oracle/src/keeper.js

```diff
@@ -0,0 +1,250 @@
+const { Networker, createReferencePriceFeedForEmp } = require("@uma/financial-templates-lib");
+const { getPrecisionForIdentifier, createObjectFromDefaultProps, MAX_UINT_VAL } = require("@uma/common");
+const { getAbi } = require("@uma/core");
+
+class OptimisticOracleKeeper {
```

Comment:
nit about this name, as mentioned in an above comment. Also, Ideally, the class name should match the name of the file. What do you think about calling it `OptimisticOracleProposer`?

Keeper might be a better name as it does both proposals, disputes and requests. if you want to keep it named as such then please match the file name to the class name.

Reply:
I renamed!